### DISCUSSION TITLE
Not able to download from Vidsrc

### DISCUSSION THREAD

**Discussion by trevshep** [aclap-dev/video-downloadhelper#1383] [Help and Q&A]
### Your question

Hi,

I have been using VDH for Vidsrc ever since I found it. Whenever I try to download something this past weekend from vidsrc it says download failed. I didn't see any recent threads about vidsrc so couldn't tell if it's a known issue or not. Having same issue on both the stable and beta release. Tried it on Chrome as well with the same issue.

Thanks!

<img width="1915" alt="Untitled" src="https://github.com/user-attachments/assets/58929d3d-c2f8-40ac-94d5-0658404505e1">

### Addon & CoApp version and details

version: 9.1.0.41
target: mozilla
channel: beta
lang: en-US
coapp: {"found":true,"path":"D:\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"d3fbd3fb93134f19"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:131.0) Gecko/20100101 Firefox/131.0
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "download_directory": "C:\\Users\\tbol9\\dwhelper",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive"
        }
    },
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 7,
    "successfull_dl": 45,
    "never_show_successfull_dl_message": true
}

**Comment by Mattmjs** [comment]
Where is it happening on what site ?

**Reply by trevshep** [reply]
[Vidsrc.net/embed/tt10954652](https://vidsrc.net/embed/tt0073195) is an example.

**Reply by Mattmjs** [reply]
The example doesn't play so the source file might not exist. Is there something playable you can provide.

**Reply by trevshep** [reply]
Yeah that's been an issue as well this weekend with an error popping up at the top of the screen. File does exist and plays sometimes takes a few tries to get it to play

**Reply by GrampaWildWilly** [reply]
That URL https://vidsrc.net/embed/tt0073195 is for the movie Jaws.  When I went to the page, VDH recognized nothing.  So I clicked the Play button & it did not play.  It just showed another Play button.  However . . .

![image](https://github.com/user-attachments/assets/f5c6ec70-aea4-4def-892c-b0f6207192c7)

That looked encouraging, so I tried to download it.  That just gave me the annoying error message asking me to report the problem.  I clicked the Play button a second time & the movie did begin to play in the web page.

Then I decided to look at the Network Monitor.  There was an HLS master manifest as well as 2 HLS stream manifests.  I ran all 3 URLs through ffprobe.  That got me 3 errors saying 403 Forbidden.  This appears to be one of those stealth DRM situations, rather ironically appropriate for that movie.

But this is stealth DRM, not real DRM.  There is no DRM protection on this.  You can see in my image above that the DRM badge does not appear at the top of the Firefox window to the left of the URL.  As a test, I used OBS to record about a minute of this.  It recorded fine, with audio.

I will say that this web site seems very sluggish.  The audio was muted by default.  I clicked the little speaker icon to unmute it but nothing happened.  So I clicked it again a few times & discovered that it was not true that nothing happened.  It just had a lag of about 5 seconds from the time I clicked the icon until the audio actually unmuted.  I found the same annoying lack of responsiveness on the settings gear & the fullscreen control.  They all worked but reluctantly.  In any case, if you really want this, you can use a screen recorder.  You'll have to let it record for about 2 hours & you'll have to be careful about what else you do on your computer at the same time.  But you can record this.

**Reply by imperfectax** [reply]
So is it impossible now to download something from Vidsrc due to the stealth DRM?

**Reply by GrampaWildWilly** [reply]
Well, obviously I haven't looked at every movie on the site.  In fact, I'd never visited the site before your problem report appeared.  So the only thing I can say for sure is that the manifests for Jaws are blocked from access without some sort of secret procedure which is of course not published.  Sites are not necessarily consistent.  You might be able to use VDH to download some content but not other content.  I wouldn't know any more than you would.  We're both just users.  I don't work for VDH.  I'm just trying to help a fellow user.  Same as Matt.
